Please Consider this Pull Request - Updated to match latest Cedar

FergalMohan opened the pull request:
Updated to use CocoaPods, XCode7 and XCTestCase
Fergal Mohan (EMC MobEx)

Commit:
Updated to match latest Cedar

Updated to use CocoaPods, XCode7 and XCTestCase

akitchen commented:
Hi there - could you please add the pods dir to .gitignore?

Commit:
Per Request from Andrew

add the pods dir to .gitignore

FergalMohan commented:
Hi Andrew, done and committed to the Pull Request,
Fergal

akitchen commented:
Hello @FergalMohan  - please remove the pods dir from the commit as well, and squash into one commit.  Then we should be good to go.  Since Cedar is brought in via cocoapods, we shouldn't be seeing its files here.  Thanks!

Commit:
Revert "Updated to match latest Cedar"

This reverts commit 65f056a9284fc7099a15d1df0d50f51ec5f64af4.

Commit:
Make a single commit that has all the changes

All the changes excluding the Pods folder. Added some notes to the
Readme to explain how to get that folder

FergalMohan commented:
Hi Andrew, i revert the commit that had the included the Pods and redid the changes without that step. I added some comments to the end of the readme to explain how to get the Pods folder. I think you should be good to go now with a merge that will bring this Repo up to date with the latest stuff. Let me know, Fergal
